config/locales/ja.yml

```
@@ -1,65 +1,85 @@
 ---
 ja:
   activerecord:
+    models:
+      AdminUser: "管理者"
+      AwsText: "AWSテキスト"
+      Movie: "動画"
+    attributes:
+      AdminUser:
+        email: "Eメール"
+        current_sign_in_at: "最新ログイン日時"
+        sign_in_count: "ログイン回数"
+        created_at: "作成日時"
+      AwsText:
+        title: "タイトル"
+        content: "コンテンツ"
+        created_at: "作成日時"
+        updated_at: "更新日時"
+      Movie:
+        title: "タイトル"
+        url: "URL"
+        created_at: "作成日時"
+        updated_at: "更新日時"
```

モデル名はスネークケースで表さなければなりません。

```suggestion
    models:
      admin_user: "管理者"
      aws_text: "AWSテキスト"
      movie: "動画"
    attributes:
      admin_user: &user
      aws_text:
        title: "タイトル"
        content: "コンテンツ"
        created_at: "作成日時"
        updated_at: "更新日時"
      movie:
        title: "タイトル"
        url: "URL"
        created_at: "作成日時"
        updated_at: "更新日時"
```